package.json

```diff
@@ -10,11 +10,22 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@fontsource/roboto": "^5.0.12",
     "@fortawesome/fontawesome-free": "^6.5.2",
+    "@mui/icons-material": "^5.15.15",
+    "@mui/material": "^5.15.15",
+    "@types/material-ui": "^0.21.17",
+    "axios": "^1.6.8",
     "bootstrap": "^5.3.3",
```

Comment:
We shouldn't be using bootstrap as I mentioned in the session, as it has a different design system and all functionalities that we need are in MUI.